lib/exavier/cover.ex

```diff
@@ -51,7 +52,9 @@ defmodule Exavier.Cover do
           {:error, :non_existing} ->
             Logger.error("Could not find module #{non_default_module_name} defined in option :test_files_to_modules for #{test_file}.")
 
-          {:ok, _} -> {:ok, :compiled}
+          {:ok, _} ->
+            source = non_default_module_name.__info__(:compile)[:source]
```

Comment:
Nice, I didn't know about `Module.__info__/1`! 😳 

Do you think it'd be worth having an abstraction over this repetition? E.g., 

```elixir
# line 36
source = module_name |> source_path()

# ...

# line 56
source = non_default_module_name |> source_path()

defp source_path(module) do
  module.__info__(:compile)[:source]
end
```

Does it make sense? 🤔 💭

Reply:
Sure. But I try to keep PRs as concise as possible. I would be happy to do this in the next one

Reply:
Ok, fair. 😉